react-events/my-app/src/App.js

```diff
@@ -0,0 +1,30 @@
+import './App.css';
+import CustomButton from './CustomButton';
+
+function App() {
+  let handleCustomClick = (text) => {
+    window.alert(text);
+  };
+
+  return (
+    <div className="App">
+      <CustomButton
+        text={'First Button'}
+        style={{ color: 'red' }}
```

Comment:
We shouldn't be setting the `color` of the text, we should be setting the `background-color` of the button